README.md

```diff
@@ -94,7 +95,7 @@ node index.js init-near-fun-token
 Now start the services that will relay the information between the chains:
 ```bash
 node index.js start eth-relay
-node index.js start near-relay
+node index.js start near-relay --eth-master-sk 0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501201
```

Comment:
I think this should be fine to omit, as eth sk is in config.json?

Reply:
This is a different key. If we try using the same secret key for different concurrent operations (like relay and transfer) they will conflict through the nonces and one of them will fail.

Reply:
Got it